packages/lib/src/components/internal/Countdown/A11yManager.ts

```diff
@@ -0,0 +1,73 @@
+import { interpolateElement } from '../../../language/utils';
+
+export class A11yManager {
+    protected TRANSLATION_KEY = 'sr.wechatpay.timetopay';
+    protected LONG_TIMEOUT = 180_000;
+    protected MID_TIMEOUT = 60_000;
+    protected SHORT_TIMEOUT = 30_000;
+
+    private srPanel;
+    private srInterval;
+    private i18n;
+    private timeout;
+    private timeLeft;
+
+    constructor(props) {
+        const { srPanel, i18n } = props;
+        this.srPanel = srPanel;
+        this.i18n = i18n;
+        // Force the srPanel to update arialRelevant
+        this.srPanel.update({ arialRelevant: 'additions text' });
```

Comment:
I wonder if updating the component (which then unmounts & remounts the components is the best way here?
Could you also pass the `ariaRelevant` value as a prop when declaring the `SRPanelProvider` and have the Provider set the value into the SRPanel?
Just wondering...